modules/light-clients/08-wasm/types/misbehaviour_handle_test.go

```diff
@@ -184,18 +168,12 @@ func (suite *TypesTestSuite) TestCheckForMisbehaviour() {
 				Data: []byte{1},
 			}
 
+			clientStore := suite.chainA.App.GetIBCKeeper().ClientKeeper.ClientStore(suite.ctx, endpoint.ClientID)
```

Comment:
suite.chainA.GetContext()?

Reply:
ah, nice! missed this one! we also have https://github.com/cosmos/ibc-go/issues/4949 to catch any ones that are missed.